play_by_play.py

```diff
@@ -1,60 +1,26 @@
 import pandas as pd
 import numpy as np
-import time
 import config
-from connect import Connector
-from nba_api.stats.endpoints import playbyplayv2
 from games import Games
 
 
 class PlayByPlay:
 
     columns = config.play_by_play_columns
 
-    def __init__(self, game_ids):
-        self.game_ids = game_ids
+    def __init__(self, raw_plays_dfs):
+        self.raw_plays_dfs = raw_plays_dfs
 
-    def get_plays(self):
+    def clean_and_return_plays(self):
```

Comment:
Same comment as before. Why not make `self._clean_data()` public and call that. You can rename accordingly, but this is redundant as this public function only calls the private function